plugins/gcalendar/.eslintrc.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  extends: [require.resolve('@backstage/cli/config/eslint')],
+};
```

Comment:
```suggestion
module.exports = require('@backstage/cli/config/eslint-factory')(__dirname);
```

this is the new style used by module-declaring packages